strawberryfields/gbs/similarity.py

```diff
@@ -19,10 +19,33 @@
 
 .. currentmodule:: strawberryfields.gbs.similarity
 
-Functionality for calculating feature vectors of graphs using GBS.
+This module provides tools to measure how similar graphs are using GBS. Measuring similarity is
+achieved by mapping GBS samples from each graph to a feature vector and calculating the overlap
+between these vectors.
 
-Summary
--------
+The functionality here is based upon the research papers:
+:cite:`bradler2018graph,schuld2019quantum,bradler2019duality`.
+
+An accompanying tutorial can be found `here <??>`.
+
+Coarse-graining GBS samples
+---------------------------
+
+GBS feature vectors can be composed of the probabilities of certain events from sampling. These

```

Comment:
How about "GBS feature vectors can be composed of a coarse-grained combination of elementary samples." ?